### DISCUSSION TITLE
How to disable "history sharing" ?

### DISCUSSION THREAD

**Discussion by belotte55** [starship/starship#2400] [Q&A] [answered]
Hi guys !

I work with many terminal (iTerm2 on macOS) sessions opened.
When i use the up arrow to re-use a previous command, i find command from other sessions.
Is it because of Starship. I had not this issue before.
Is it possible to disable this behaviour ?

Thx !

**Comment by davidkna** [comment, marked as answer]
Starship does not change any shell history settings.
If you are using the default `zsh` shell you should be able disable history sharing by adding `unsetopt share_history` to your `.zshrc`.

**Reply by belotte55** [reply]
`unsetopt share_history` effectively disabled this behaviour. Thank you very much !
